include/event2/dns_compat.h

```diff
@@ -211,6 +211,108 @@ int evdns_resolve_ipv4(const char *name, int flags, evdns_callback_type callback
 EVENT2_EXPORT_SYMBOL
 int evdns_resolve_ipv6(const char *name, int flags, evdns_callback_type callback, void *ptr);
 
+/**
```

Comment:
Let's not add compat layer for new functionality

Reply:
Agree